php/libraries/SinglePointLogin.class.inc

```diff
@@ -245,7 +245,7 @@ class SinglePointLogin
      *                         If false, it won't provide any output
      *                         in those situation.
      *
-     * @return bool true if the username and password are valid
+     * @return bool False unless account is valid and password correct.
```

Comment:
on line 240, comment for $password should be changed from username to password

Reply:
done